vocode/streaming/transcriber/base_transcriber.py

```diff
@@ -3,18 +3,22 @@
 import asyncio
 import audioop
 from abc import ABC, abstractmethod
-from typing import Generic, Optional, TypeVar, Union
+from typing import TYPE_CHECKING, Generic, Optional, TypeVar, Union
 
 from vocode.streaming.models.audio import AudioEncoding
 from vocode.streaming.models.transcriber import TranscriberConfig, Transcription
 from vocode.streaming.utils.speed_manager import SpeedManager
 from vocode.streaming.utils.worker import AbstractWorker, AsyncWorker, ThreadAsyncWorker
 
+if TYPE_CHECKING:
```

Comment:
Not familiar with this pattern. Why wouldn't we just import StreamingConversation all the time? Is it for build performance?

Reply:
Circular imports. We import only for type checking but at runtime it doesn't actually import